### ISSUE TITLE
Add H1 to the storybook Component Page

### ISSUE THREAD

**Issue by frastlin** [storybooks/storybook#5001]
Hello,
Can the main components screen please get an h1 (heading level 1) before the iframe example? The content of the h1 should be the name of the selected component.
[This is the page I am talking about](http://react.carbondesignsystem.com/?selectedKind=Accordion&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel)
Adding a heading will fulfill [WCAG 2.4.6 Headings and Labels.](https://www.w3.org/WAI/WCAG21/quickref/#headings-and-labels)
Before a screen reader user knows how to navigate, they need to press the down arrow key the number of times there is a component. For Carbon, that is around 60 times every time you wish to see a new component.

**Comment by igor-dv** [comment]
I think it will be possible with the version 5, @ndelangen, fix me if I am wrong
